src/js/components/award/table/TransactionsTable.jsx

```diff
@@ -110,7 +110,7 @@ export default class TransactionsTable extends React.Component {
             const isLast = i === tableMapping.table._order.length - 1;
 
             const displayName = tableMapping.table[column];
-            let columnWidth = Math.max(measureTableHeader(displayName),
+            let columnWidth = Math.max(measureTableHeader(displayName) - 40,
```

Comment:
recommend defining a variable outside the definition of the class called `columnWidthOffset` and setting it to 40. That way it's a little easier to read.

Reply:
fixed